packages/webpack-configs/src/dev.ts

```diff
@@ -134,14 +128,8 @@ export function defineDevConfig(swcConfig: SwcConfig, options: DefineDevConfigOp
             publicPath: publicPath ?? `${https ? "https" : "http"}://${host}:${port}/`
         },
         cache: cache && {
-            type: "filesystem",
-            allowCollectingMemory: true,
-            maxMemoryGenerations: 1,
-            cacheDirectory: cacheDirectory,
-            // Took from https://webpack.js.org/configuration/cache/#cachebuilddependencies.
-            buildDependencies: {
-                config: [__filename]
-            }
+            type: "memory",
+            maxGenerations: 1
         },
         // (ACTUALLY NOT FIXING ANYTHING AT THE MOMENT)
         // Fixes caching for environmental variables using the DefinePlugin by forcing
```

Comment:
Is the snapshot section still required? If you prefer, this question can be answered after the PR is merge.

Reply:
Technically there's nothing wrong with keeping them and the builds will work fine as-is. But I'm looking at the default values, and I believe the addition of the extra hash checks in some of these would slow the rebuilds (probably not by much though). Regardless, I think it's best to remove these and let the defaults take over. I'll update the branch.